build/assets/license/header_yaml.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 The Kubernetes Dashboard Authors.
```

Comment:
We keep here license template files used by gulp tasks to automatically update file headers. You would need to update `check-license-headers` and `update-license-headers` gulp tasks. Right now it is not used.

Reply:
Yea, I just needed a place for copy&paste manually a couple of files

Reply:
Should the file be removed?

Reply:
If it won't be used by our build pipeline then yes.